t04_refactored.py

```diff
@@ -182,7 +183,105 @@ def team_11_adv():
 
 ###################################################################################
 
+
 def team_12_adv():
+    """https://docs.google.com/document/d/13xlcbGqW_CVr6wpsi9CH2nEjf8Qk6_S0cyVs3417i_I/edit?usp=sharing
+    Baella Morgan, Ariana Meatchem, Jaren Vessels"""
+    global HasSword
+    global goblindead
+    # Intro
+    print("You stand in your makeshift campsite, a pitiful fire smoulders, sending black smoke to the sky.")
+    sleep(delay)
+    print("To the North: The Mystic Wood lies, fabeled to hold a great treasure.")
+    sleep(delay)
+    print("To the South: The Mouth of a cave, you can hear noises creep from within.")
+    sleep(delay)
+    print("To the West: A long road that you came from.")
+    sleep(delay)
+    print("To the East: A vast meadow of yellow flowers.")
+    sleep(delay)
+    ############
+    direction = input("Choose a direction or action[Search the camp/North/South/East/West]: ")
+    # CHECK FOR VALID INPUT###
+    while direction != "north" and direction != "south" and direction != "east" and direction != "west" and direction != "search camp":
```

Comment:
Your `input` line asks for "South" but your conditional checkes for "south", which aren't the same. An easy fix is to drop everything to lower case using `.lower()`:

`while direction.lower() != "north" ...`